docs/DeveloperGuide.adoc

```diff
@@ -1260,6 +1276,223 @@ Use case ends.
 +
 Use case resumes at step 1.
 
+[discrete]
+=== Use case: Creating a class
+
+*Pre-condition*
+
+Module code exists in data file.
+
+*MSS*
+
+1.  User enters command to create classroom.
+2.  Classroom is created for the module.
+3.  System displays message of successful creation of class.
+
+*Extensions*
+
+[none]
+* 1a. User entered invalid command.
++
+[none]
+** 1a1. System shows ‘invalid format’ error.
++
+Use case resumes at step 1.
+
+[discrete]
+=== Use case: Deleting a class
+
+*Pre-condition*
+
+Module code exists in data file.
+
+*MSS*
+
+1.  User enters command to delete a class from module.
+2.  Classroom is deleted from module.
+3.  System displays message of successful deletion of class from module.
+
+
```

Comment:
There's an extra newline here that can be removed (after MSS and before Extensions)